command/tests/test_identify.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+from nose.tools import nottest
+import command.identify as id
+
+class TestGetOperatorArtifactType(TestCase):
+    @nottest
+    def id_test(self, file, expected ):
+        yaml = open(file).read()
```

Comment:
I recommend context manager to have file properly closed:
```python
with open(fname) as f:
    data = f.read()
```

`file` is also built-in and should't be used

Reply:
No longer using the built-in `file` variable.